### PR TITLE
Refactor to target .NET standard 1.1; unified naming conventions.

### PR DESCRIPTION
**Pull request by 71**
Suggestion given in #5 implemented. Additionally, some types and files were renamed to better follow the .NET naming conventions: `MIPS*` -> `Mips*`, `x86*` -> `X86*`.

### CODE REVIEW

File: src/Unicorn.Net/MIPS/MIPSRegisters.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ﻿namespace Unicorn.MIPS
```
Comment (FICTURE7) [inline src/Unicorn.Net/MIPS/MIPSRegisters.cs]: This should be `Unicorn.Mips`.

**Comment by FICTURE7** [comment]
I've been trying to make my mind up about the naming conventions but I think this should be the naming convention to use.

**Comment by 71** [comment]
I don't think I missed anything, besides some samples or file names.

**Comment by FICTURE7** [comment]
Looks good to me 🎉

**Pull request merged**